docs/guides/engineering/fresh-config.md

```diff
@@ -0,0 +1,78 @@
+# Ensuring Fresh Feature Flag Configurations
+
+When working with feature flags, there are certain use cases, like kill switches, where it can be important to ensure your application always has the most up-to-date configuration. This guide explains how to maintain fresh configurations across different platforms when using Eppo's SDKs.
+
+## Server-Side Applications
+
+Server-side applications automatically handle configuration freshness through built-in polling mechanisms. You can configure the polling interval during initialization of the SDK.
+
+## Client-Side Applications
+
+### JavaScript Client
+For browser-based applications, you have several options to ensure fresh configurations:
+
+1. **Enable Polling**
+```javascript
+const client = new EppoClient({
+    apiKey: 'your-api-key',
+    pollInterval: 30000 // Poll every 30 seconds
+});
+```
+
+2. **Manual Refresh**
+```javascript
+// Force a configuration refresh when needed
+await eppoClient.load();
+```
+
+### Mobile Applications
+Mobile apps can implement similar strategies:
+
+1. **Configure Polling**
+```swift
+// iOS Example
+let client = EppoClient(
+    apiKey: "your-api-key",
+    pollInterval: 30 // seconds
+)
+```
+
+```kotlin
+// Android Example
+val client = EppoClient.builder()
+    .apiKey("your-api-key")
+    .pollInterval(30) // seconds
+    .build()
```

Comment:
for android and kotlin you'd use `.forceReinitialize(true)` to do a pushed reset

method is here: https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/main/java/cloud/eppo/android/EppoClient.java#L195